students/Lincoln_Zhang/Session3/mailroom.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import sys
+
+print("Mailroom")
+
+donors = [("Fred Flintstone", [100, 200, 50]),
+          ("Barney Rubble", [100, 50, 600]),
+          ("Wilma Flintstone", [1000, 50, 600, 200]),
+          ("Pebbles Flintstone", [10, 5, 6]),
+          ]
+
+def Thank_You():
+    cash = []
+    name = input("Please tell me your name: ")
+    cash_input = int(input("Please tell me how much you would like to donate: "))
+    cash.append(cash_input)
```

Comment:
what if the donor already is in the list?

Reply:
fixed in part-2